pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pybusinesscentral"
-version = "3.2.0"
+version = "3.3.0"
```

Comment:
Might be worth having a stab at swapping to uv if you wanna have a crack.

Here's some inspo:
I did it w pymyob a few months back: https://github.com/uptick/pymyob/pull/105
Raphael just did it w bronski: https://github.com/uptick/bronski/pull/7#pullrequestreview-3083356913